src/presentation/components/KYC/KYC.tsx

```diff
@@ -63,6 +63,7 @@ export const KYC = () => {
               name="country"
               error={errors.country}
               options={countries}
+              alpha3Code
```

Comment:
rename this prop if previous change is applied

Reply:
Done!